dlp/src/main/java/com/example/dlp/QuickStart.java

```diff
@@ -78,7 +77,7 @@ public static void main(String[] args) throws Exception {
               .build();
       ContentItem contentItem = ContentItem.newBuilder().setByteItem(byteContentItem).build();
 
-      String projectId = ServiceOptions.getDefaultProjectId();
+      String projectId = "";
```

Comment:
nit: should a comment be added here to instruct the user?

Reply:
This file needs to be cleaned up yet - `ServiceOptions` seems to have been removed and it won't compile currently - I'll move it into an overloaded function in a separate PR.